qClassify3DMASCDialog.cpp

```diff
@@ -140,70 +139,109 @@ void Classify3DMASCDialog::writeSettings()
 	settings.setValue("keepAttributes", keepAttributesCheckBox->isChecked());
 }
 
-void Classify3DMASCDialog::setCloudRoles(const QList<QString>& roles, QString corePointsLabel)
+void Classify3DMASCDialog::setComboBoxIndex(QMap<QString, QString> rolesAndNames, QLabel* label, QMap<QString, QVariant> namesAndUniqueIds, QComboBox* comboBox)
```

Comment:
```suggestion
void Classify3DMASCDialog::setComboBoxIndex(const QMap<QString, QString>& rolesAndNames, QLabel* label, const QMap<QString, QVariant>& namesAndUniqueIds, QComboBox* comboBox)
```

Reply:
otherwise the maps will be copied each time!

Reply:
Obviously, sorry